notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<p>My final comment is more of a question. You have the function to determine the status of a single test, right? So, what now? Do you have to create a statistic for thefinal test we are running now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Indeed, now I am in the process of generalizing this for set of tests throughout the night. And at the same time, settle on the details of the PASS/FAlL criteria, which we have not converged on 100%.